src/main/scala/com/databricks/labs/overwatch/pipeline/Schema.scala

```diff
@@ -208,6 +208,15 @@ object Schema extends SparkSessionWrapper {
         StructField("ssh_public_keys", StringType, nullable = true),
         StructField("single_user_name", StringType, nullable = true),
         StructField("resourceId", LongType, nullable = true),
+        StructField("cluster_size", StringType, nullable = true),
+        StructField("min_num_clusters", LongType, nullable = true),
+        StructField("max_num_clusters", LongType, nullable = true),
+        StructField("auto_stop_mins", LongType, nullable = true),
+        StructField("spot_instance_policy", StringType, nullable = true),
+        StructField("enable_photon", BooleanType, nullable = true),
+        StructField("channel", StringType, nullable = true),
+        StructField("enable_serverless_compute", BooleanType, nullable = true),
+        StructField("warehouse_type", StringType, nullable = true),
```

Comment:
good job -- I think you got all the fields you're using. Triple check to make sure they're all here AND that none of them are duplicated.